scripts/generate-action-profiles.mjs

```
@@ -0,0 +1,43 @@
+ * Elgato manifest's `Profiles` array (order preserved).
+ */
+export function buildProfilesData(manifest) {
+  const profiles = Array.isArray(manifest.Profiles) ? manifest.Profiles : [];
+  return profiles.map((p) => ({ name: p.Name, deviceType: p.DeviceType }));
+}
+
+function main() {
+  const manifest = JSON.parse(readFileSync(MANIFEST_FILE, "utf-8"));
+  const data = buildProfilesData(manifest);
+  writeFileSync(OUTPUT_FILE, JSON.stringify(data, null, 2) + "\n", "utf-8");
+  console.log(`Generated ${OUTPUT_FILE}`);
```

ESLint

[error] 'console' is not defined.

(no-undef)